README.md

```diff
@@ -1,13 +1,118 @@
 # terraform-example-foundation
-Example repo showing how the CFT modules can be composed to build a foundation
+This is an example repo showing how the CFT terraform modules can be composed to build a secure GCP foundation. The supplied structure and code is intended to form a starting point for building your own foundation with pragmatic defaults you can customize to meet your own requirements. Currently, the code is built heavily around Cloud Build to allow teams to quickly get started without needing to deploy a CI/CD tool, although it is worth noting the code can easily be executed by your preferred tool.
```

Comment:
Two nits:
1. Usually I put each sentence on a separate line. It makes review/editing easier.
2. `Terraform` should be capitalized.

Reply:
Will fix the spelling of capitalization of Terraform. On the first point, it kind of ruins the flow of the summary, so I would prefer to keep it if you don't mind.

Reply:
Sounds good.

FYI I meant separate lines purely in the Markdown. They're still rendered as a single paragraph. Fine either way though.

Reply:
Ah cool, I will fix before merge. Did not know that